test/webapps/bookstore-spring/src/main/java/com/springapp/mvc/extensions/WebRequestRunIdTelemetryInitializer.java

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
The web app is deployed before, and regardless, the integration tests. Therefore you cannot define the initializer as part of the f/w.